k8s/scripts/kubelet-config-helper.sh

```diff
@@ -1500,7 +1534,7 @@ function main() {
 	# * RKE2: Host-based kubelet managed by rke2-agent's systemd service (Rancher's RKE2 approach).
 	# * Systemd+Docker: Docker-based kubelet managed by a systemd service (Lokomotive's approach).
 	# * Systemd: Host-based kubelet managed by a systemd service (most common approach).
-	#
+	# * Systemd: k3s when run as an agent (k3s-agent.service), if k3s is run as controlplane + node (k3s.service) it will not work
```

Comment:
Thanks; to avoid having two "systemd" entries, I would re-word as: "k3s: when run as agent only (if k3s is run as control plane + node (i.e., k3s.service) it won't work)."